### ISSUE TITLE
Update Project Web Site

### ISSUE THREAD

**Issue by Vitralius** [CankayaUniversity/ceng-407-408-2024-2025-Rule-and-Munus-AI-Based-Game-Development-Project#7]
Update project web site by considering the rubrics.

**Comment by Semihoguz17** [comment]
Website Update Completed 

✅ Added project advisors' information.  
✅ Linked group members’ LinkedIn profiles.  
✅ Completed missing GitHub links for better accessibility.  
✅ Improved navigation by adding color variations to the menu.  
✅ Uploaded a demo video on the homepage for a better user experience.
